test_globaltimer is flaky

This test failed in CI on my PR but I see this locally as well:
```
        @triton.jit
        def kernel(Out1, Out2):
            start = tl.extra.cuda.globaltimer()
            off = tl.arange(0, 128)
            for i in range(100):
                tl.store(Out1 + off, tl.load(Out1 + off) + 1)
            end = tl.extra.cuda.globaltimer()
            tl.store(Out2, end - start)
    
        out1 = to_triton(np.zeros((128,), dtype=np.int64), device='cuda')
        out2 = to_triton(np.zeros((1,), dtype=np.int64), device='cuda')
        h = kernel[(1,)](out1, out2)
>       assert out2[0] > 0
E       AssertionError: assert tensor(0, device='cuda:0') > 0
```

Running in a loop:

```
$ for i in `seq 1 100` ; do if python3 -m pytest test/unit/language/test_core.py -k test_globaltimer > /dev/null 2>&1 ; then echo "Pass" ; else echo "Fail"; fi; done
Fail
Pass
Pass
Pass
Pass
Pass
Pass
Pass
Fail
Fail
Pass
Fail
Pass
Pass
Pass
Pass
Pass
Pass
Pass
Pass
Fail
Fail
Fail
Pass
Pass
Pass
Pass
```

--- a/python/test/unit/language/test_core.py
+++ b/python/test/unit/language/test_core.py
@@ -2838,7 +2838,7 @@ def test_globaltimer():
     def kernel(Out1, Out2):
         start = tl.extra.cuda.globaltimer()
         off = tl.arange(0, 128)
-        for i in range(100):
+        for i in range(10000):
             tl.store(Out1 + off, tl.load(Out1 + off) + 1)
         end = tl.extra.cuda.globaltimer()
         tl.store(Out2, end - start)

